### PR TITLE
Add /version handler which exports gitVersion

### PR DESCRIPTION
**Pull request by leoleovich**
```
go run -ldflags "-X main.gitCommit=123" github.com/leoleovich/3djuggler
```
```
curl localhost:8888/version
123
```

### CODE REVIEW

**Review by abulimov — CHANGES_REQUESTED** [review]
Overall looks really good, but please add build instructions explaining how to build with real commit as a version to the README.md, and even maybe a Makefile to simplify the process.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
+### /pause
+Pause the job
+### /cancel
+Cancel the job
+### /reshedule
+Give more time before jobs gets marked as "timed out"
+### /version
+In order to use this functionality, you needs to compile juggler like this:
+```
+go run -ldflags "-X main.gitCommit=123" github.com/leoleovich/3djuggler
```
Comment (abulimov) [inline README.md]: This runs juggler, not builds it.
I suggest using something like this instead:
```
export GIT_COMMIT=$(git rev-list -1 HEAD) && \
  go build -ldflags "-X main.gitCommit=$GIT_COMMIT" github.com/leoleovich/3djuggler
```

**Review by abulimov — CHANGES_REQUESTED** [review]
I still think having makefile is better, but even without it doc should tell how to build, not run with gitCommit.

**Review by yrk-lab — COMMENTED** [review]
👍🏻

**Pull request merged**